frontend/components/bridge/EstimatedCompletionTime.tsx

```diff
@@ -14,6 +17,30 @@ const EstimatedTimeRow = styled(Flex)`
   }
 `;
 
+const CountUp = () => {
+  const [seconds, setSeconds] = useState(0);
+
+  useEffect(() => {
+    const timer = setInterval(() => {
+      setSeconds((prev) => prev + 1);
+    }, 1000);
```

Comment:
it's not accurate to use 1000 here and expect it to be called exactly in 1 second, depending on the code base it can be much later. might be fine because we're not an alarm/timer app, but jsyk https://blog.bitsrc.io/how-to-get-an-accurate-setinterval-in-javascript-ca7623d1d26a